### ISSUE TITLE
Startup some one GameBar widget by .net program(WPF)

### ISSUE THREAD

**Issue by TaiwanSpring** [microsoft/XboxGameBarSamples#108]
**Is your feature request related to a problem? Please describe.**
I want to open the GameBar widget by WPF (maybe on button click)

**Describe the solution you'd like**
We can open GameBar menu by Win key + G key now.
So I think that can provide some window api like OpenWidget(Id) to open widget .

**Describe alternatives you've considered**
I can use keyboard hook send Win key + G key to open GameBar menu only, but It is not enough.

**Comment by vidager** [comment]
Yes, you can do that with the URI scheme.   Details here (Scroll to URI scheme section): https://docs.microsoft.com/en-us/gaming/game-bar/api/xgb-widgetcontrol#methods

From a WPF app, you just need to call one of the shell APIs that can launch an app via shell execute.  Two options would be
 1) from C/C++ or .NET via pinvoke: ShellExecute/ShellExecuteEx.  
 2) from any language with WinRT access (C#, C++, etc):  https://docs.microsoft.com/en-us/uwp/api/windows.system.launcher.launchuriasync?view=winrt-22000

If you need help figuring out the URI for the widget you want to launch, let us know.  Keep in mind the 'launch' command starts Game Bar and activates the widget you specify, but it won't bring it game bar "on screen".  

If you need to bring game bar "on screen", you can use the 'launchForeground' command instead of 'launch'.  Please do this instead of keyboard hooks.  Also, please don't ever bring game bar to the visual foreground ("on screen") without an authentic user action (such as a click). It sounds like this is what you're planning anyway so that's good.  We want to avoid game bar coming "on screen" without the user wanting it to. 

Thanks,
Ben

**Comment by TaiwanSpring** [comment]
Hello Ben
Can I find the "PackageFamilyName" and "AppId" in app store dashboard =>Product management => Product identity 

![image](https://user-images.githubusercontent.com/27178589/148716077-a71dbb2f-6a07-487f-bfed-0f2e1961d419.png)

Thank you.
